test/models/project_test.rb

```diff
@@ -0,0 +1,26 @@
+require 'test_helper'
+
+class ProjectTest < ActiveSupport::TestCase
+  def setup 
+    @user = build(:user)
+    @subject = Project.create(name: 'test', slug: 'this-should-be-unique', user: @user)
```

Comment:
Suggest using FactoryBot, e.g. `@subject = build(:project, name: 'test', user: @user)` and testing `slug` generation rather than hard coding.